AdyenDropIn/Components/PreApplePayComponent.swift

```diff
@@ -92,7 +92,15 @@ internal final class PreApplePayComponent: PresentableComponent,
 extension PreApplePayComponent: PaymentComponentDelegate {
     
     internal func didSubmit(_ data: PaymentComponentData, from component: PaymentComponent) {
-        delegate?.didSubmit(data, from: self)
+        let checkoutAttemptId = component.context.analyticsProvider.checkoutAttemptId
+        let updatedData = PaymentComponentData(paymentMethodDetails: data.paymentMethod,
+                                               amount: data.amount,
+                                               order: data.order,
+                                               storePaymentMethod: data.storePaymentMethod,
+                                               browserInfo: data.browserInfo,
+                                               checkoutAttemptId: checkoutAttemptId,
+                                               installments: data.installments)
+        delegate?.didSubmit(updatedData, from: self)
```

Comment:
this should probably use the `submit` function, right?

Reply:
Yes, you're right! Let me update it

Reply:
Done!